.github/ISSUE_TEMPLATE/operate-feature.md

```diff
@@ -56,6 +51,22 @@ tasklist:alpha3-->
 - Release version ( in which version this fixed/feature will be released)
 <!-- Add here -->
 
-
-### :green_book: Link to the test case 
+### Link to the test case 
```

Comment:
@ArezooAzar can you please explain what the purpose of this section is? Do we still need it?

Reply:
Thanks. I will reword the explanation that this is explicitly for any QA-created test cases and not for links to test cases in the code (those are discoverable through the linked PRs).